pkg/cli/cvd.go

```diff
@@ -247,6 +252,39 @@ func (c *cvdCreator) createWithOpts() ([]*hoapi.CVD, error) {
 	return res.CVDs, nil
 }
 
+func (c *cvdCreator) createCVDFromLocalArtifacts() ([]*hoapi.CVD, error) {
+	fmt.Println("here createCVDFromLocalArtifacts")
+	srcs, err := loadLocalArtifacsSrcs(c.opts.LocalArtifacts)
+	if err != nil {
+		return nil, fmt.Errorf("failed loading local artifacts file: %w", err)
+	}
+	if err := srcs.validate(); err != nil {
+		return nil, fmt.Errorf("invalid local artifacts: %w", err)
+	}
+	uploadDir, err := c.service.HostService(c.opts.Host).CreateUploadDir()
```

Comment:
From this part, I think all logics are same as `createCVDFromLocalBuild`. Could we unite logic with introducing a new function?

Reply:
true, however let's have the refactoring in a future PR as I'm working on changes on the upload API to handle compression before uploads.

Reply:
Okay. Sounds good. Just let this comment thread be opened. Leaving a TODO comment might be better, to grab refactoring chance by someone in the future.